mcp_extension/snyk_tools.json

```diff
@@ -5,6 +5,7 @@
       "description": "Run a SCA test on project dependencies to detect known vulnerabilities. Use this to scan open-source packages in supported ecosystems like npm, Maven, etc. Supports monorepo scanning via `--all-projects`. Outputs vulnerability data in JSON if enabled.",
       "command": ["test"],
       "standardParams": ["all_projects"],
+      "requiresTrust": true,
```

Comment:
just an idea - should we maybe invert this setting? That way, the default is to require trust, instead of the other way round.